Elissa/Classes/UIViewControllerExtension.swift

```diff
@@ -37,13 +38,26 @@ extension UIViewController {
     
     private func updatePresentingFrame(_ sourceView: UIView) -> UIView {
         var sourceFrame = sourceView.frame
-        
         if let height = tabBarController?.tabBar.frame.size.height {
             sourceFrame.origin.y = UIScreen.main.applicationFrame.size.height - height / 2
         }
+        return UIView(frame: sourceFrame)
+    }
+}
+
+extension UIView {
+    public func showElissa(fromSourceView sourceView: UIView, configuration: ElissaConfiguration, onTouchHandler: (() -> Void)?) -> Elissa {
```

Comment:
Eventually add discardable result annotation? I had a warning when presenting it.